core-ui/src/components/Predefined/Create/Gateways/Forms/TlsForm.js

```diff
@@ -6,13 +6,51 @@ import {
   TLS_VERSIONS,
   validateTLS,
   isTLSProtocol,
+  isHTTPProtocol,
 } from './../helpers';
 import { ResourceForm } from 'shared/ResourceForm';
 import { K8sResourceSelectWithUseGetList } from 'shared/components/K8sResourceSelect';
 import * as Inputs from 'shared/ResourceForm/inputs';
 
+const HttpTlsForm = ({ server, servers, setServers }) => {
+  const handleHttpsRedirect = () => {
+    if (!server?.tls) {
+      server['tls'] = {
+        httpsRedirect: true,
+      };
+    } else {
+      server.tls = undefined;
+    }
+  };
+
+  const { t } = useTranslation();
+  return (
+    <ResourceForm.CollapsibleSection
+      title={t('gateways.create-modal.advanced.tls.tls')}
+      resource={server}
+      defaultOpen={true}
+      setResource={() => setServers([...servers])}
+    >
+      <ResourceForm.FormField
```

Comment:
shouldn't you remove the corresponding switch from TSL/HTTPS form as well? especially since you removed (well, renamed) the i18n entries for it?

Reply:
For sure, I forgot